docs/modules/datasources/pages/entity_datasource.adoc

```diff
@@ -1,8 +1,8 @@
-= Entities datasource
+= Entities Datasource
 
 == General
 
-Use the _Entities Datasource_ to retrieve entity model objects from OpenNMS.
+Use the Entities Datasource to retrieve entity model objects from OpenNMS.
```

Comment:
Also, in the Helm context, I think it's ok to use OpenNMS as shorthand to refer to Horizon or Meridian. Since Helm is built outside of the main docs, using a variable either won't work, or might be too complicated.

Reply:
Variable wouldn't work, as it's outside that context, and Helm applies to either.  Would have to be `Horizon/Meridian` if not `OpenNMS`.